resources/definitions/z-bolt_plus.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "Z-Bolt Plus",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Z-Bolt",
+        "manufacturer": "Z-Bolt Co.",
+        "file_formats": "text/x-gcode",
+        "platform": "z-bolt_plus_platform.stl",
+        "machine_extruder_trains":
+        {
+            "0": "z-bolt_plus_extruder_0"
```

Comment:
This extruder is the same for both printers, so you could consider to only use two references to the same extruder definition (less to maintain if they are always the same)